### ISSUE TITLE
Let metaflac output to a different file

### ISSUE THREAD

**Issue by kalrish** [xiph/flac#75]
**TL;DR**: metaflac should support outputting to a different file, like [`vorbiscomment(1)`](https://www.systutorials.com/docs/linux/man/1-vorbiscomment/).

**Background**: I have created a music compilation system which works kinda like a software build system. When possible, it encodes and tags in different steps so that a mere tag change doesn't result in the reencoding of the entire library. Unfortunately, metaflac doesn't support outputting to a different file (or to stdout), so I have to copy the untagged files manually.

Currently:
```
flac -o song-untagged.flac song.wav
cp -- song-untagged.flac song.flac
metaflac --set-tag=TITLE=Song song.flac
```

Desired:
```
flac -o song-untagged.flac song.wav
metaflac --set-tag=TITLE=Song -o song.flac song-untagged.flac
```

Since metaflac is designed to operate on multiple files simultaneously and we don't want to break the interface, I think the easiest would be to add a `-o` option. If it is specified, only one input file may be passed; any more should cause an error.
